tests/functional/modules/test_zos_copy_func.py

```diff
@@ -1575,6 +1575,47 @@ def test_copy_pds_to_existing_pds(ansible_zos_module, args):
         hosts.all.zos_data_set(name=dest, state="absent")
 
 
+@pytest.mark.pdse
+def test_copy_pds_member_with_system_symbol(ansible_zos_module,):
+    """This test is for bug #543 in GitHub. In some versions of ZOAU,
+    datasets.listing can't handle system symbols in volume names and
+    therefore fails to get details from a dataset.
+    """
+    hosts = ansible_zos_module
+    # The volume for this dataset should use a system symbol.
+    # This dataset and member should be available on any z/OS system.
+    src = "SYS1.SAMPLIB(IZUPRM00)"
```

Comment:
I tried this on one of our EC's, and volume is '*****'.  I will assume this means it can't process the volume name.  Nice!